README.md

```diff
@@ -15,7 +15,8 @@ syntax highlighting for stan cells.
 - Stan language syntax highlighting in all cells beginning with `%%stan`
 - Compile a stan model and save it as a pystan variable by running a `%%stan` cell
 - No longer worry about `model_code`, reading in stan files, etc.
-
+- Support for caching compiled stan models: re-executing a cell with the same code will 
+  not recompile the code but uses a cached version.
```

Comment:
I'm a bit worried the caching is a bit obfuscated from the user so would appreciate some info on this in the README:

```suggestion
  not recompile the code but uses a cached version. Cached models are stored your home directory,
  e.g. `~/.cache/stan` on Unix systems (see
  [`pathlib.Path.home`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.home) for how
  to find it).
```